### DISCUSSION TITLE
Problem with reading from mic

### DISCUSSION THREAD

**Discussion by DamianosMk** [pschatzmann/ESP32-A2DP#549] [Q&A] [answered]
Hello, I am using an esp32 and an SPH0645. Does this library support the SPH0645? If it does, I use the same connections?

**Comment by pschatzmann** [comment, marked as answer]
As far as I can see this is a regular I2S microphone, so I don't see any reason why this should not work.
But you will need to figure out yourself if there are some quirks...

Just read the [corresponding AudioTools Wiki](https://github.com/pschatzmann/arduino-audio-tools/wiki/External-ADC)

**Reply by DamianosMk** [reply]
Thanks!
